modules/tinymce/src/themes/silver/test/ts/browser/editor/EventsTest.ts

```diff
@@ -0,0 +1,75 @@
+import { Mouse, UiFinder, Waiter } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyContentActions, TinyDom, TinyHooks, TinyUiActions } from '@ephox/mcagar';
+import { SugarBody } from '@ephox/sugar';
+import { assert } from 'chai';
+
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.themes.silver.editor.EventsTest', () => {
+  const hook1 = TinyHooks.bddSetup({
+    base_url: '/project/tinymce/js/tinymce',
+    plugins: 'link',
+  }, [ Theme ]);
+
+  const hook2 = TinyHooks.bddSetup({
+    base_url: '/project/tinymce/js/tinymce',
+    plugins: 'link',
+  }, [ Theme ]);
```

Comment:
```suggestion
  const settings = {
    base_url: '/project/tinymce/js/tinymce',
    plugins: 'link',
    menubar: 'file'
  };

  const hook1 = TinyHooks.bddSetupLight(settings, [ Theme ]);
  const hook2 = TinyHooks.bddSetupLight(settings, [ Theme ]);
```
This removes the duplication and also saves us having to render UI components we're not using in the tests.